index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Conference Page - home</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,400;0,700;0,900;1,400&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav class="nav-bar">
+      <a class="mobile menu-h "><img class='mobile menu-h' src="Icons/menu.png"></a>
+      <div class="nav-bar-div1">
+        <ul class="nav-bar-div1-ul">
+          <li class="nav-bar-div1-li"><img class="img-nav-bar-fb" src="Icons/facebook.png"></li>
```

Comment:
You have done well implementing the changes requested by the previous code reviewer. While I was reviewing your work I noticed something when I change the screen size to mobile view I noticed the links in the black Navigation menu are visible in list form please it would be nice if you corrected this error because tho social media links are not supposed to be visible in your mobile view


This is how your Webpage looks in mobile view:

![Screenshot 2023-02-08 at 5 29 06 PM](https://user-images.githubusercontent.com/79506105/217576901-3b0224e2-08d2-4ba3-b4bd-36eebd5bb782.png)
Kindly fix this issue.

Reply:
Thank you